packages/dapi/test/unit/grpcServer/handlers/blockheaders-stream/subscribeToBlockHeadersWithChainLocksHandlerFactory.spec.js

```diff
@@ -0,0 +1,186 @@
+const { ChainLock } = require('@dashevo/dashcore-lib');
+const { expect, use } = require('chai');
+// eslint-disable-next-line no-underscore-dangle
+const _sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const dirtyChai = require('dirty-chai');
+const chaiAsPromised = require('chai-as-promised');
```

Comment:
`expect` available globally. Sinon is available with `this.sinon`. Please check `lib/test/bootstrap.js`

Reply:
Fixed